public/index.html

```diff
@@ -3,8 +3,6 @@
   <head>
     <meta charset="utf-8">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
-    <link rel='stylesheet' href='https://cdn.rawgit.com/twbs/bootstrap/48938155eb24b4ccdde09426066869504c6dab3c/dist/css/bootstrap.min.css'>
-    <link rel='stylesheet' href='%PUBLIC_URL%/style.css'>
```

Comment:
It still works? I thought sass gets compiled to this file

Reply:
I'm importing the sass in the index.js file instead

Reply:
If it works thats fine